src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2593,20 +2592,42 @@ components:
         role_key:
           $ref: "#/components/schemas/RoleKey"
         assignment:
-          type: object
+          $ref: "#/components/schemas/ComponentEntityRoleAssignmentEntryAssignment"
+        updater_roles:
+          type: array
+          items:
+            $ref: "#/components/schemas/RoleKey"
+
+    ComponentEntityRoleAssignmentEntryAssignment:
+      type: object
+      required:
```

Comment:
MINOR: Whilst we're here, is it easy to add a description?

Resolving the rule for this role:
* If it says `Explicit`, the rule can be read off the `explicit_rule` property on this object.
* If it's `Owner`, the rule can be read off `role_assignments.owner.rule` off the role assignments object two levels up.